app/screens/home/channel_list/categories_list/categories/index.ts

```diff
@@ -3,14 +3,14 @@
 
 import {withDatabase} from '@nozbe/watermelondb/DatabaseProvider';
 import withObservables from '@nozbe/with-observables';
-import {of as of$} from 'rxjs';
+import {of as of$, combineLatest} from 'rxjs';
```

Comment:
Use combineLatestWith instead, check the docs to understand the difference, but mainly it wmits once all values are set at least once